aws-amp/adot/README.md

```diff
@@ -0,0 +1,123 @@
+# Kubecost with AWS Distro for Open Telemetry
+
+## Overview
+
+This guide will walk you through the steps to deploy Kubecost with AWS Distro for Open Telemetry (ADOT) to collect metrics from your Kubernetes cluster.
+
+## Prerequisites
+
+Follow AWS guide to install the ADOT daemonset: <https://docs.aws.amazon.com/prometheus/latest/userguide/AMP-onboard-ingest-metrics-OpenTelemetry.html>
+
+Update all configuration files in this folder that contain `YOUR_*` with your values.
+
+This guide assumes that the Kubecost helm release name and the Kubecost namespace are equal, which allows a global find and replace on `YOUR_NAMESPACE`.
+
+## Architecture Diagram
+
+![Architecture Diagram](multi-cluster-prometheus-kubecost-architecture.png)
+
+## Configuration
+
+1. Update all configuration files with your cluster name (replace all `YOUR_CLUSTER_NAME_HERE`). The examples use the key of `cluster` for the cluster name. You can use any key you want, but you will need to update the configmap and deployment files to match. A simplified version of the ADOT DS installation is below.
+
+
+### ADOT Configuration
+
+Deploy the ADOT Daemonset, there are many options for this. At a minimum, Kubecost needs the provided scrape config to be added to the ADOT Prometheus configmap.
+
+1. Add the Kubecost scrape config to the ADOT Prometheus configmap:
+
+    Note that this sample configmap also contains cadvisor metrics, which is required by Kubecost.
+
+    ```bash
+    kubectl apply -f example-configs/prometheus-daemonset.yaml
+    ```
+
+Alternatively, you can add these items to your [existing configmap](example-configs/kubecost-adot-scrape-config.yaml)
+
+
+### Kubecost Agent Installation
+
+1. Create the Kubecost namespace:
+
+    ```bash
+    kubectl create ns YOUR_NAMESPACE
+    ```
+
+1. Create the AWS IAM policy to allow Kubecost to query metrics from AMP:
+
+    ```bash
+    aws iam create-policy --policy-name kubecost-read-amp-metrics --policy-document file://iam-read-amp-metrics.json
+    ```
+
+1. Create the AWS IAM policy to allow Kubecost to write to the `federated-store` S3 bucket:
+
+    ```bash
+    aws iam create-policy --policy-name kubecost-bucket-policy --policy-document file://iam-kubecost-metrics-s3-policy.json
+    ```
+
+1. Configure the Kubecost Service Account:
```

Comment:
This is assuming OIDC is configured and we are using IRSA. Can we call that out somewhere and link to say the AWS docs on configuring OIDC/IRSA?